test/parallel/test-util-inspect.js

```diff
@@ -3436,3 +3436,13 @@ assert.strictEqual(
     '[Function: Symbol(f)]',
   );
 }
+
+{
+  const error = new Error();
+  error.stack = [Symbol('foo')];
+
+  assert.strictEqual(
+    inspect(error),
+    '[[\n  Symbol(foo)\n]]'
```

Comment:
Could we change the `improveStack()` method to result in:

```
Error: message
[\n
  Symbol(foo)\n
]
```

I think that would be beneficial. Otherwise it's going to be very tricky to understand what the output is about.

Reply:
Unfortunately, it seems like assigning `.name` stringifies it immediately, so we can't alter rendering behavior based on that later. Thoughts?